components/LandingPage/sections/Cre8orsWay.tsx

```diff
@@ -0,0 +1,35 @@
+import Image from "next/image"
+import { useMediaQuery } from "usehooks-ts"
+import SectionTitle from "../SectionTitle"
+import SectionContent from "../SectionContent"
+
+import ProfileImage from "../../../public/assets/Landing/profile.png"
+
+const Cre8orsWay = () => {
+  const match490 = useMediaQuery("(max-width: 490px)")
```

Comment:
again, what is this `490` breakpoint? Can you rename this variable to be more clear?